tests/unit/test_mutation.py

```
@@ -0,0 +1,174 @@
+from typing import Optional, List, LiteralString
```

Ruff (0.11.9)

`typing.LiteralString` imported but unused

Remove unused import: `typing.LiteralString`

(F401)